launch-helper.ps1

```diff
@@ -50,10 +56,51 @@ elseif ($optionSelected -eq "4"){
   . "$scriptPath\update-chart-versions.ps1" -v development | Tee-Object -file "$scriptPath\logs\$optionSelected-Outputlogs.txt"
 }
 elseif ($optionSelected -eq "5"){
-  kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0/aio/deploy/recommended.yaml
-  start-sleep 10
-  write-output "<---------STARTING SCRIPT---------->`n"
-  . "$scriptPath\start-dashboard-proxy.ps1" | Tee-Object -file "$scriptPath\logs\$optionSelected-Outputlogs.txt"
+  write-output "`n--- Advanced Local Launch ---"   
+  $optionSelected2=Read-Host -Prompt "Q: Do you want to download Remote Files from Azure Container Repository (Yes/No)"
+  
+  while("yes","no","Yes","No","YES","NO","y","n","N" -notcontains $optionSelected2)
+  {
+    write-host "Answer not recognised..." 
+    $optionSelected2 = Read-Host "Q: Do you want to download Remote Files from Azure Container Repository? (Yes/No)"
+  }
+
+  if ("no","No","NO","n","N" -contains $optionSelected2){
+    write-output "<---------STARTING SCRIPT---------->`n"
+    . "$scriptPath\launch-or-update-local.ps1" -r $false | Tee-Object -file "$scriptPath\logs\$optionSelected-Outputlogs.txt" -Append
+  }
+  else {
+    $optionSelected3=Read-Host -Prompt "Q: Do you want to download updated versions of the Charts? (Yes/No)"
+    
+    while("yes","no","Yes","No","YES","NO","y","n","N" -notcontains $optionSelected3)
+    {
+      write-host "Answer not recognised..." 
+      $optionSelected3 = Read-Host "Q: Do you want to download updated versions of the Charts? (Yes/No)"
+    }
+
+    if ("no","No","NO","n","N" -contains $optionSelected3){
```

Comment:
There should be a way to check the a normalised input matches a smaller list of options